zulipterminal/model.py

```diff
@@ -906,6 +906,34 @@ def topics_in_stream(self, stream_id: int) -> List[str]:
 
         return list(self.index["topics"][stream_id])
 
+    def _fetch_stream_email_from_endpoint(self, stream_id: int) -> Optional[str]:
+        """
+        Fetch stream's email address with specified stream_id and
+        returns stream_email_address as a String.
```

Comment:
Continuing from the other docstrings comment,
By looking only at the function definition line,
we know that the function fetches stream email, takes a stream_id as input, and returns a string optionally.
In that case, we wouldn't necessarily have to repeat it in the docstrings. If there are some latent aspects that could be overlooked, they could be mentioned instead on their own, like the server version, ZFL, etc you've mentioned.

I know it's commonly known that having docstrings is good, but for ZT, I think we're following the convention that we add docstrings only where necessary, or useful, which it might not be so for every function.

This is my interpretation.

Reply:
Most methods in `model.py` have docstrings, so I added one. But now that you mention it, I guess it can be removed. What do you think, @neiljp?